pkg/util/spectool/spectool.go

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (c) 2017, MegaEase
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package spectool
```

Comment:
propose to rename this package, as it is used not only for specs.

Reply:
How about `codectool`?

Reply:
`codec` often refers to video/audio compress/decompress, maybe `encoding` or just `jsonyaml`?

Reply:
Nope, I propose this name because it is a normal name in golang: https://github.com/ugorji/go#go-codec , which I've used before.

Reply:
Okay, I'm fine with it.

Reply:
Updated